source/llassetgen/source/Image.cpp

```diff
@@ -72,15 +73,44 @@ namespace llassetgen {
         assert(ft_bitDepth == bitDepth);
 
         stride = size_t(std::ceil(float(getWidth() * bitDepth) / 8));
+        size_t paddingBytes = padding / 8;
+
+        /* because there are (padding % 8) padding bits, each byte from the original bitmap is
```

Comment:
There is no such thing as left and right, please refer to them by significance (LSBs and MSBs).